src/lib/components/TagTypeSelection.svelte

```
@@ -0,0 +1,45 @@
+	let { tagtypes = $bindable(), ...rest } = $props();
+
+	const selectedTagtypes = $derived(
+		tagtypes?.length
+			? tagtypeList
+					.filter((tagtype) => tagtypes.includes(String(tagtype.id)))
+					.map((tagtype) => tagtype.name)
+			: 'Select tagtypes'
+	);
+</script>
+
+{#snippet myBadge(children: any)}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)